adapters/acuityads/acuityads.go

```diff
@@ -0,0 +1,193 @@
+package acuityads
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"text/template"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/macros"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type AcuityAdsAdapter struct {
+	endpoint template.Template
+}
+
+func NewAcuityAdsBidder(endpointTemplate string) *AcuityAdsAdapter {
+	template, err := template.New("endpointTemplate").Parse(endpointTemplate)
+	if err != nil {
+		return nil
```

Comment:
Please add a glog call to let the host know if there was an error. This will be improved in the near future.